src/annotator/util/test/scroll-test.js

```diff
@@ -128,5 +128,36 @@ describe('annotator/util/scroll', () => {
         delete document.body.tagName;
       }
     });
+
+    // A test for scrolling when there is a target within the 'details' tag.
```

Comment:
I think this comment is probably not needed, as it's the same as the test title.

```suggestion
```

Reply:
Thank you for your suggestion. I have applied it right away.